co2eq/purchase/footprints.yml

```diff
@@ -5,1685 +5,2708 @@
 #
 ---
 _children:
-  Car parts and services:
+  FOOD AND NON-ALCOHOLIC BEVERAGES:
+    icon: shopping-cart
+    intensityKilograms:
+      AT: 0.5343799565570979
+      AU: 0.9728272152026072
+      BE: 0.7309710456582461
+      BG: 0.7101497680609757
+      BR: 1.5683241050582177
+      CA: 1.0199525915022476
+      CH: 0.49706032489409785
+      CN: 2.2935901921453525
+      CY: 0.9955099939851199
+      CZ: 0.9636715695236164
+      DE: 0.6505547690588043
+      DK: 0.817390437852872
+      EE: 1.4090507334725522
+      ES: 0.5735894582156583
+      FI: 0.49482483030098173
+      FR: 0.6177632719273307
+      GB: 0.5983709842857473
+      GR: 0.8184213218917619
+      HR: 0.6953397580397929
+      HU: 0.8954240374882119
+      ID: 1.0257234527403039
+      IE: 0.9963156652109284
+      IN: 1.8848752434917917
+      IT: 0.6116844186338382
+      JP: 0.7958752906183161
+      KR: 0.8488610127405243
+      LT: 1.8095424274114351
+      LU: 1.203833917823845
+      LV: 1.3308501439644416
+      MT: 0.458862163898547
+      MX: 1.447213449395304
+      NL: 0.7966504751312278
+      NO: 0.7200394390458047
+      PL: 0.995905241143346
+      PT: 0.7793799478733805
+      RO: 0.685345049055374
+      RU: 1.0496705712750907
+      SE: 0.4015730836995318
+      SI: 0.7073367263212595
+      SK: 0.8658500455716434
+      TR: 1.03296626866704
+      TW: 0.9188054181996063
+      US: 1.5364118088902992
+      WA: 1.7350952874168
+      WE: 2.5882699304742083
+      WF: 2.911992173797467
+      WL: 1.5390818481438304
+      WM: 1.653008860728267
+      ZA: 2.2718077885896837
+    oldKeys:
+      - PURCHASE_CATEGORY_FOOD_SUPERMARKET
+      - Grocery store
+    source: https://github.com/tmrowco/bloom-contrib/tree/master/co2eq/purchase/exiobase
+    unit: EUR
+    coicopCode: '01'
```

Comment:
The codes are a mix of strings and not-strings - think this should be aligned :)
![image](https://user-images.githubusercontent.com/3296643/102368571-47bdfe00-3fbb-11eb-8787-5e4e4a1ad887.png)

Reply:
This is legacy, but indeed, seems like now is the good time to get rid of this!

I was thinking the same about the discrepancy between all-caps and normal categories name but this might apparently this is the consequence of the parsing of the EXIOBASE csv. As we are not parsing anything on a regular basis, it might be worth incorporating too and to keep in mind if we ever need to re-parse, thoughts?

Reply:
Ah sorry, thought you had updated the values here. Github sucks at diffing this kind of changes 😅 
Yeah, not sure why there's a difference, but that would require migrating all activities as well, so it's a slightly bigger task.

We can do this at a later point when/if we actually use the coicopCode then :)

Reply:
Just a quick note here: it might be worth updating the python scripts in the `purchase` folder to force them to create string coicopCode instead of mixed types. The reasoning is that we might re-use that script to update the values later on, so it would be good to keep things consistent!

Reply:
Yes indeed, Unfortunately we can't uniformise everything to remove the ', as for example 08 would be parsed as an int, worth 8. For consistency, it would thus probably be better to keep ticks everywhere.

Reply:
Ticks everywhere make sense. :)